charts/rancher-csp-adapter/Chart.yaml

```diff
@@ -13,4 +13,4 @@ annotations:
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux
   catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
-  catalog.cattle.io/kube-version: '>= 1.22.0-0 < 1.27.0-0'
+  catalog.cattle.io/kube-version: '>= 1.22.0-0 < 1.28.0-0'
```

Comment:
Shouldn't it be `catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.28.0-0'`

Reply:
As far as I know, the 1.23 min version is applicable for v2.8 release and not for the v2.7. Michael can confirm this.

Reply:
Typically we avoid dropping support for a minor version on a patch version. That being said, reviewing the [support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-9/) 1.23 appears to be the lowest supported version so it might be best to update this to bring this in line with the rest of the 2.7 line.